reference/7.3/Microsoft.PowerShell.Core/About/about_Logging_Non-Windows.md

```diff
@@ -41,57 +135,96 @@ TIMESTAMP MACHINENAME powershell[PID]: (COMMITID:TID:CID)
 - **PID** - The process ID of the process that wrote the log entry.
 - **COMMITID** - The **git commit** ID or tag used to produce the build.
 - **TID** - The thread ID of the thread that wrote the log entry.
-- **CID** - The hex channel identifier of the log entry. 10 = Operational, 11 =
-  Analytic
+- **CID** - The hex channel identifier of the log entry.
+  - 0x10 = Operational
+  - 0x11 = Analytic
 - **EVENTID** - The event identifier of the log entry.
 - **TASK** - The task identifier for the event entry
 - **OPCODE** - The opcode for the event entry
 - **LEVEL** - The log level for the event entry
 - **MESSAGE** - The message associated with the event entry
 
-> [!NOTE]
-> **EVENTID**, **TASK**, **OPCODE**, and **LEVEL** are the same values as used when
-> logging to the Windows event log.
+**EVENTID**, **TASK**, **OPCODE**, and **LEVEL** are the same values as used
+when logging to the Windows event log.
+
+### Write PowerShell log message to a separate file
 
-### Filtering PowerShell log entries using rsyslog
+It's also possible to redirect the PowerShell log entries to a separate file.
+When the PowerShell log entries are redirected to a separate file, they're no
+longer logged to the default syslog file.
 
-Normally, PowerShell log entries are written to the default `location/file` for
-**syslog**. However, it's possible to redirect the entries to a custom file.
+The following steps configure PowerShell log entries on Ubuntu to write to a
+log file named `powershell.log`.
 
-1. Create a conf for PowerShell log configuration and provide a number that's
-   less than 50 (for `50-default.conf`), such as `40-powershell.conf`. The file
-   should be placed under `/etc/rsyslog.d`.
+1. Create a config (`conf`) file for the PowerShell log configuration in the
+   `/etc/rsyslog.d` directory using a text file editor such as `nano`. Prefix
+   the filename with a number that's less than the default. For example,
+   `40-powershell.conf` where the default is `50-default.conf`.
 
-1. Add the following entry to the file:
+   ```bash
+   sudo nano /etc/rsyslog.d/40-powershell.conf
+   ```
+
+1. Add the following information to the `40-powershell.conf` file:
 
    ```
    :syslogtag, contains, "powershell[" /var/log/powershell.log
    & stop
    ```
 
-1. Make sure `/etc/rsyslog.conf` includes the new file. Often, it will have a
-   generic include statement that looks like following configuration:
+1. Verify that `/etc/rsyslog.conf` has an include statement for the new file.
+   It may have a generic statement that includes it, such as:
 
    `$IncludeConfig /etc/rsyslog.d/*.conf`
```

Comment:
Nit: As this is a line in a text file by itself, might be better to specify as a codeblock:
``````suggestion
   ```text
   $IncludeConfig /etc/rsyslog.d/*.conf
   ```
``````